### PR TITLE
Updated envvars for python image

### PR DESCRIPTION
**Pull request by soltysh**
One more update, this time to python env variables.
@adellape @bfallonf 

// fyi @rhcarvalho

### CODE REVIEW

File: using_images/sti_images/python.adoc [hunk 1]
```diff
@@ -91,6 +91,15 @@ https://github.com/openshift/sti-python/tree/master/3.3/test/setup-test-app[setu
 available.
 
 |`*APP_CONFIG*`
-|This variable indicates the path to a module which contains a
+|This variable indicates the path to a valid Python file with
```
Comment (adellape) [inline using_images/sti_images/python.adoc]: s/with/with a/

?

**Comment by adellape** [comment]
One potential nit, otherwise LGTM.

**Comment by soltysh** [comment]
Fixed

**Comment by adellape** [comment]
Thanks, will merge after build passes.

**Pull request merged**